x/parameter/types/keys.go

```diff
@@ -36,7 +36,9 @@ const (
 
 	// USDT
 	USDT = "uusdt"
-	
+
+	ATOM = "uatom"
```

Comment:
what more coins are we going to hardcode upcoming?

Reply:
All of these are just dummy as they will be IBC denoms.

Reply:
Let's have this as params of parameter module and it will be implemented in another PR.